dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -20,7 +20,11 @@ def script_level_solved_response(response, script_level)
   end
 
   def has_another_level_to_go_to?(script_level)
-    script_level.next_progression_level
+    if script_level.script.professional_learning_course?
+      !script_level.end_of_stage?
+    else
+      script_level.next_progression_level
+    end
```

Comment:
I think @laurelfan suggested this in a previous PR but really these should be `script_level.has_another_level_to_go_to?` (on the `ScriptLevel` model) instead of `has_another_level_to_go_to?(script_level)`.